str/data_frame_creator.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error, too-many-locals, broad-exception-raised
+"""
+This script merges all the VCFs from one STR caller into a .CSV (ExpansionHunter) or .TSV (GangSTR) format that can be read into R. 
+analysis-runner --access-level test --dataset tob-wgs --description 'data frame creator' --output-dir 'hoptan-str/tob_test_crams/data_frames' data_frame_creator.py  --input-dir-eh=gs://cpg-tob-wgs-test/hoptan-str/tob_test_crams/output_calls/eh_0_based --input-dir-gangstr=gs://cpg-tob-wgs-test/hoptan-str/tob_test_crams/output_calls/gangstr_0_based --output-name-eh=eh.csv --output-name-gangstr=gangstr.tsv
+
+"""
+import click
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import remote_tmpdir, output_path
+import hailtop.batch as hb
+from google.cloud import storage
+
+config = get_config()
+
+
+def eh_csv_writer(input_dir):
+    """Creates a CSV file containing dataframe of merged EH VCFs"""
+    bucket_name, *components = input_dir[5:].split('/')
+    client = storage.Client()
+    bucket = client.bucket(bucket_name)
+    blobs = client.list_blobs(bucket_name, prefix='/'.join(components))
+    files = {f'gs://{bucket_name}/{blob.name}' for blob in blobs}
+    csv = (
+        ','.join(
+            [
+                'sample_id',
+                'chr',
+                'start',
+                'e_qual',
+                'end',
+                'repeat_units_in_ref',
+                'ref_sequence_length',
+                'motif',
+                'e_gt',
+                'e_so',
+                'e_allele_1',
+                'e_allele_2',
+                'e_repci',
+                'e_adsp',
+                'e_adfl',
+                'e_adir',
+                'e_lc',
+            ]
+        )
+        + '\n'
+    )
+
+    for file in files:
+        if file.endswith('.vcf'):
+            blob = bucket.blob(file[6 + len(bucket_name) :])
+            with blob.open('r') as f:
+                array = f.readlines()
+                for line in array:
+                    if line.startswith('##'):
```

Comment:
This whole section will work, but please be aware of VCF parsing libraries that will streamline this whole process massively, e.g. [pyvcf](https://pyvcf.readthedocs.io/en/latest/INTRO.html) or CyVCF2